opensearch/services/datasets/datasets.php

```diff
@@ -337,7 +366,14 @@ public function getQueryBody($type) {
 				$body .= "?dataset dco:yearOfPublication ?id . ";
 				$body .= "BIND(str(?id) AS ?label) . ";
 				break;
-				
+
+			case "datatypes":
```

Comment:
For facet queries the query is selecting "?id ?label (COUNT(DISTINCT ?project) AS ?count)" by default.  This means that the query body described here should use the ?id and ?label variables.  

You will need to make the following changes:
- [ ] ?d --> ?id
- [ ] ?dataType_label --> ?label